tests/jballerina-unit-test/src/test/resources/testng-jballerina.xml

```diff
@@ -1,170 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There seem to be some deviations between these files and testng.xml. Most importantly there seem to be some additional tests here. WDYT think about leaving these files as is for the moment and removing them when fixing https://github.com/ballerina-platform/ballerina-lang/issues/28770 just to make sure we're tracking all of the missed tests? 

I don't think copying the additional tests directly to testng.xml will work since some of them actually have issues - futures for example - https://github.com/ballerina-platform/ballerina-lang/pull/28786/commits/221cdee629f6824809b8711fbaa6fa9395f31ee6, https://github.com/ballerina-platform/ballerina-lang/issues/28758

Reply:
Sure will reset these